internal/access/identity.go

```diff
@@ -173,10 +173,6 @@ func UpdateIdentityInfoFromProvider(c *gin.Context, oidc providers.OIDCClient) e
 			logging.Errorf("failed to revoke invalid user session: %s", nestedErr)
 		}
 
-		if nestedErr := data.DeleteProviderUsers(db, data.ByIdentityID(identity.ID), data.ByProviderID(provider.ID)); nestedErr != nil {
```

Comment:
Sessions are still revoked, just the provider is still seen as a way that the user logged in. This matches previous behavior.